src/main/java/at/hannibal2/skyhanni/data/item/SkyhanniItems.kt

```diff
@@ -0,0 +1,175 @@
+    ENCHANTED_CACTUS_GREEN,
+    ENCHANTED_COOKIE,
+    ENCHANTED_ENDER_PEARL,
+    ENCHANTED_ENDSTONE,
+    ENCHANTED_GOLDEN_CARROT,
+    ENCHANTED_MELON,
+    ENCHANTED_OBSIDIAN,
+    ENCHANTED_PUMPKIN,
+    ENCHANTED_RED_MUSHROOM,
+    ENDERMAN,
+    ENDER_GAUNTLET,
+    ENDER_NECKLACE,
+    ENDER_PEARL,
+    ENDSTONE_GEODE,
+    END_BOOTS,
+    END_CHESTPLATE,
+    END_HELMET,
+    END_LEGGINGS,
+    END_STONE_SHULKER,
+    ETHERWARP_CONDUIT,
+    ETHERWARP_MERGER,
+    FARMING_FOR_DUMMIES,
+    FARMING_WAND,
+    FERMENTO_ARTIFACT,
+    FIFTH_MASTER_STAR,
+    FIRE_VEIL_WAND,
+    FIRST_MASTER_STAR,
+    FOURTH_MASTER_STAR,
+    FUMING_POTATO_BOOK,
+    GOLDEN_CARROT,
+    GOLDEN_DRAGON,
+    GOLD_PICKAXE,
+    GRAND_EXP_BOTTLE,
+    GREEN_THUMB,
+    HEAVY_PEARL,
+    HELIX,
+    HIGH_CLASS_ARCHFIEND_DICE,
+    HOT_POTATO_BOOK,
+    ICE_SPRAY_WAND,
+    INFERNO_GENERATOR_1,
+    INFINI_VACUUM,
+    INFINI_VACUUM_HOOVERIUS,
+    JACOBS_TICKET,
+    JALAPENO_BOOK,
+    KISMET_FEATHER,
+    KUUDRA_BURNING_TIER_KEY,
+    KUUDRA_FIERY_TIER_KEY,
+    KUUDRA_HOT_TIER_KEY,
+    KUUDRA_INFERNAL_TIER_KEY,
+    KUUDRA_TIER_KEY,
+    LARVA_HOOK,
+    LEATHER,
+    MAGIC_RUNE,
+    MAGMA_FISH,
+    MANA_DISINTEGRATOR,
+    MAP,
+    MITE_GEL,
+    NETHER_STALK,
+    NEW_BOTTLE_OF_JYRRE,
+    NEW_YEAR_CAKE,
+    PAPER,
+    PEST_BEETLE_MONSTER,
+    PEST_CRICKET_MONSTER,
+    PEST_EARTHWORM_MONSTER,
+    PEST_FLY_MONSTER,
+    PEST_LOCUST_MONSTER,
+    PEST_MITE_MONSTER,
+    PEST_MOSQUITO_MONSTER,
+    PEST_MOTH_MONSTER,
+    PEST_RAT_MONSTER,
+    PEST_SLUG_MONSTER,
+    PLASMA,
+    POLARVOID_BOOK,
+    PUMPKIN,
+    RABBIT_HIDE,
+    RACING_HELMET,
+    RANCHERS_BOOTS,
+    RECOMBOBULATOR_3000,
+    RED_MUSHROOM,
+    REFINED_BOTTLE_OF_JYRRE,
+    REVENANT_GENERATOR_1,
+    ROUGH_AMBER_GEM,
+    ROUGH_AMETHYST_GEM,
+    ROUGH_AQUAMARINE_GEM,
+    ROUGH_CITRINE_GEM,
+    ROUGH_JADE_GEM,
+    ROUGH_JASPER_GEM,
+    ROUGH_ONYX_GEM,
+    ROUGH_OPAL_GEM,
+    ROUGH_PERIDOT_GEM,
+    ROUGH_RUBY_GEM,
+    ROUGH_SAPPHIRE_GEM,
+    ROUGH_TOPAZ_GEM,
+    SECOND_MASTER_STAR,
+    SEEDS,
+    SHADOW_FURY,
+    SHRIMP_THE_FISH,
+    SILENT_DEATH,
+    SIL_EX,
+    SKELETON_MASTER_CHESTPLATE,
+    SKYBLOCK_ENIGMA_SOUL,
+    SKYMART_HYPER_VACUUM,
+    SKYMART_TURBO_VACUUM,
+    SKYMART_VACUUM,
+    SORROW,
+    SQUASH_RING,
+    STARRED_SHADOW_FURY,
+    STONK_PICKAXE,
+    SUPERBOOM_TNT,
+    SUSPICIOUS_SCRAP,
+    TARANTULA_GENERATOR_1,
+    THE_ART_OF_PEACE,
+    THE_ART_OF_WAR,
+    THIRD_MASTER_STAR,
+    TITANIC_EXP_BOTTLE,
+    TRANSMISSION_TUNER,
+    TURBOMAX_VACUUM,
+    ULTIMATE_REITERATE,
+    VAMPIRE_GENERATOR_1,
+    VOIDLING_GENERATOR_1,
+    VOLTA,
+    VOODOO_DOLL_WILTED,
+    WEIRDER_TUBA,
+    WEIRD_TUBA,
+    WHEAT,
+    WISP_POTION,
+    WIZARD_PORTAL_MEMENTO,
+    WOOD_SINGULARITY,
+    ZOMBIE_COMMANDER_WHIP,
+    ZOMBIE_KNIGHT_SWORD,
+    ZOMBIE_SOLDIER_CUTLASS,
+    ;
+
+    // TODO make it work with items with a dash in their name
```

Comment:
should have an optional consturctor/paramter with its id, and when we do the dyes instead of doing INK_SACK-2 for the ID it should be the actual color ie. CACTUS_GREEN_DYE